ccd-definition/ComplexTypes/HearingBookingDetails.json

```diff
@@ -0,0 +1,98 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingBookingDetails",
```

Comment:
I don't think we need the word 'details'. This could just be a `hearingBooking` object

Reply:
Agreed :)